dspace-server-webapp/src/test/java/org/dspace/app/rest/builder/CollectionBuilder.java

```diff
@@ -155,6 +155,11 @@ public Collection build() {
 
     @Override
     public void cleanup() throws Exception {
+        context.turnOffAuthorisationSystem();
+        if (collection.getAdministrators() != null) {
+            Group adminGroup = collection.getAdministrators();
+            collectionService.removeAdministrators(context, collection);
+        }
```

Comment:
Please update this to create its own Context, similar to `deleteWorkflowGroups()`. I've run into occasional Hibernate issues with reusing the global context for deletion of test data.  The best practice is the method used in `deleteWorkflowGroups()` which uses a `try` to open a new context, then deletes & completes.